src/messages.ts

```diff
@@ -83,8 +89,15 @@ export function buildWalkthroughMessage(summary: PullRequestSummary): string {
     message += `| \`${escapedPath}\` | ${escapedSummary} |\n`;
   }
 
+  const payload = {
+    commits: commits,
+  };
```

Comment:
Consider defining an interface or type for the payload object structure. This would improve type safety and make it easier to maintain the code as the payload structure evolves:
```typescript
interface WalkthroughPayload {
  commits: string[];
}

const payload: WalkthroughPayload = {
  commits: commits,
};
```